Extensions/timestamps.js

```diff
@@ -224,12 +253,88 @@ XKit.extensions.timestamps = new Object({
 
 			var timestamp = responseData.posts[0].timestamp;
 			date_element.html(this.format_date(moment(new Date(timestamp * 1000))));
-			date_element.removeClass("xtimestamp_loading");
+			date_element.removeClass("xtimestamp-loading");
 			XKit.storage.set("timestamps", "xkit_timestamp_cache_" + post_id, timestamp);
 		})
 		.catch(() => this.show_failed(date_element));
 	},
 
+	react_add_timestamps: function() {
+		var posts = $("[data-id]:not(.xkit_timestamps)");
+
+		if (posts.length === 0) {
+			return;
+		}
+
+		XKit.extensions.timestamps.check_quota();
+
+		posts.each(function() {
+			var post = $(this);
+			post.addClass("xkit_timestamps");
+
+			var post_id = $(this).attr("data-id");
+
+			var xtimestamp_class = "xtimestamp";
+			if (XKit.extensions.timestamps.in_search && !$("#search_posts").hasClass("posts_view_list")) {
+				xtimestamp_class = "xtimestamp-in-search";
+			}
```

Comment:
does `#search_posts` or the `.posts_view_list` class even exist on the react search page?

Reply:
I definitely can't find it, I'll just remove that part.